modules/ROOT/pages/batch-processing-concept.adoc

```diff
@@ -25,10 +25,19 @@ If you are already familiar with batch processing in Mule 3.x, you can get an ov
 
 == Overview
 
-Within a Mule application, polling for the input) to process them in chunks.
+Within a Mule application, batch processing provides a construct for
+asynchronously processing larger-than-memory data sets that are split
+into individual records. Batch jobs allow for the description of a reliable
```

Comment:
I found this sentence confusing ```Batch jobs allow for the description of a reliable
process that automatically splits up source data and stores it into persistent
queues, which can process large data sets while providing reliability``` but it could be due to the tech terminology. I would recommend to split this sentence into two.

Reply:
done